src/gpu/intel/jit/gemm/generator/strategy_parser.cpp

```diff
@@ -561,6 +561,13 @@ void parseStrategy(const char *str, HW hw, const GEMMProblem &problem, GEMMStrat
 
     if (strategy.persistent && !strategy.linearOrder()) strategy.cWalkOrder = WalkOrder::SimpleLinear;
 
+    // Use new LSC messages on Xe2+
+    if (hw >= ngen::HW::Xe2) {
+       strategy.A.newDP = strategy.A_prefetch.newDP = true;
+       strategy.B.newDP = strategy.B_prefetch.newDP = true;
+       strategy.C.newDP = strategy.C_prefetch.newDP = true;
+    }
+
```

Comment:
Can't believe we missed this until now :)

Reply:
It was recently move up, but not quite far enough. 😄 Not sure why it wasn't causing failures until I tried adding this kernel, though.